### PR TITLE
AZP: change roce server for io_demo to instinct

### PR DESCRIPTION
**Pull request by avildema**
## What
move io_demo run to instinct

## Why ?
roce server is slow

## How ?
change server

### CODE REVIEW

**Comment by yosefe** [comment]
@avildema need to change SSH target in 2 places in io_demo.yml:
1. ssh service 
```
      - task: SSH@0
        inputs:
          sshEndpoint: ucx-roce-client
```

2. run_io_demo.sh
```
      - bash: |
          set -eEx
          # set UCX environment variables
          export UCX_SOCKADDR_CM_ENABLE=y
          # TODO get hostname of 'ucx-roce-client' SSH service endpoint, and run on it
          $(workspace)/../test/apps/iodemo/run_io_demo.sh \
              -H swx-rdmz-ucx-roce-01,swx-rdmz-ucx-roce-02 \
```

**Comment by yosefe** [comment]
@avildema the net device name should be `bond0`
```
swx-rdmz-instinct01 : 1 servers,  0 clients
swx-rdmz-instinct02 : 0 servers,  1 clients
swx-rdmz-instinct01: Device "mlx5_bond0" does not exist.
pdsh@swx-rdmz-instinct01: swx-rdmz-instinct01: ssh exited with exit code 1
swx-rdmz-instinct02: Device "mlx5_bond0" does not exist.
pdsh@swx-rdmz-instinct01: swx-rdmz-instinct02: ssh exited with exit code 1

Error: failed to collect host IP addresses for mlx5_bond0
```

**Comment by avildema** [comment]
> @avildema the net device name should be `bond0`
> 
> ```
> swx-rdmz-instinct01 : 1 servers,  0 clients
> swx-rdmz-instinct02 : 0 servers,  1 clients
> swx-rdmz-instinct01: Device "mlx5_bond0" does not exist.
> pdsh@swx-rdmz-instinct01: swx-rdmz-instinct01: ssh exited with exit code 1
> swx-rdmz-instinct02: Device "mlx5_bond0" does not exist.
> pdsh@swx-rdmz-instinct01: swx-rdmz-instinct02: ssh exited with exit code 1
> 
> Error: failed to collect host IP addresses for mlx5_bond0
> ```

I know, currently in progress

**Comment by yosefe** [comment]
@avildema can you pls fix conflicts & push to rerun tests?

**Comment by avildema** [comment]
> @avildema can you pls fix conflicts & push to rerun tests?

Of course, will do today

**Comment by yosefe** [comment]
@avildema we still have error from corrupter, but it's not a blocker
```
2020-11-24T07:48:02.160432369Z Running ./buildlib/az-network-corrupter.sh initial_delay=20 cycles=100 downtime=5 uptime=20...
2020-11-24T07:48:02.161905961Z Initial delay 20 sec
2020-11-24T07:48:22.169937587Z #1 Put it down! And sleep 5
2020-11-24T07:48:22.224625283Z 
2020-11-24T07:48:22.226978818Z Error: unsupported server
```

can you pls squash?

**Comment by avildema** [comment]
> @avildema we still have error from corrupter, but it's not a blocker
> 
> ```
> 2020-11-24T07:48:02.160432369Z Running ./buildlib/az-network-corrupter.sh initial_delay=20 cycles=100 downtime=5 uptime=20...
> 2020-11-24T07:48:02.161905961Z Initial delay 20 sec
> 2020-11-24T07:48:22.169937587Z #1 Put it down! And sleep 5
> 2020-11-24T07:48:22.224625283Z 
> 2020-11-24T07:48:22.226978818Z Error: unsupported server
> ```
> 
> can you pls squash?

problem with access to switch, currently try to fix

**Review by yosefe — APPROVED** [review]

File: buildlib/az-helpers.sh [hunk 1]
```diff
@@ -38,6 +38,22 @@ function azure_log_issue() {
...
+# Report an error message to Azure pipeline
+function azure_log_error() {
+    test "x$RUNNING_IN_AZURE" = "xno" && return
+    msg=$1
+    set +x
+    echo "##vso[task.logissue type=error]${msg}"
+}
+
+# Report an warning message to Azure pipeline
+function azure_log_error() {
```
Comment (yosefe) [inline buildlib/az-helpers.sh:50]: rename to azure_log_warning

Comment (avildema) [reply buildlib/az-helpers.sh]: done

**Review by yosefe — APPROVED** [review]

**Pull request merged**